govdelivery-subscribe/error/index.html

```diff
@@ -11,13 +11,13 @@
 {% block content_main %}
     <h1><span class="cf-icon cf-icon-error-round"></span> Not just yet!</h1>
     <p class="short-desc">
-        We’re still building the email signup function for <br>
-        this beta release. It will be available soon. For now:
+        We couldn’t complete your signup request. Try<br>
+        again in a little while. For now:
     </p>
     <ol class="short-desc">
         <li>Keep exploring <a href="http://beta.consumerfinance.gov">beta.consumerfinance.gov</a>.</li>
-        <li>Check back often to follow our progress.</li>
-        <li>Sign up for general CFPB updates at <a href="http://www.consumerfinance.gov">www.consumerfinance.gov</a>.</li>
+        <li>Check back often to follow our progress and find new sources of information.</li>
+        <li><a href="/contact-us/"></a>Let us know</a> if there’s a particular line of work you want to hear about regularly.</li>
```

Comment:
Extra closing tag pushes text outside of the link:
`<a href="/contact-us/"></a>Let us know</a>` > `<a href="/contact-us/">Let us know</a>`